pkgs/development/libraries/utf8cpp/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, fetchFromGitHub, cmake }:
+
+stdenv.mkDerivation rec {
+  pname = "utf8cpp";
+  version = "3.1.1";
+
+  src = fetchFromGitHub {
+    owner = "nemtrif";
+    repo = "utfcpp";
+    rev = "v${version}";
+    sha256 = "1d5p9l27mxhsmm4zlxjsbxngsmr5yxrzi66x550bf5g92sj52ffc";
+  };
+
+  cmakeFlags = [
+    "-DCMAKE_BUILD_TYPE=None"
+    "-DCMAKE_INSTALL_LIBDIR=lib"
+    "-DUTF8_TESTS=OFF"
+  ];
+
+  enableParallelBuilding = true;
```

Comment:
This isn't required, nix figures this out automatically. From the nixpkgs manual:
`Unless set to false, some build systems with good support for parallel building including cmake, meson, and qmake will set it to true.`